demo/python/remote-control/server/enclave_serve.py

```diff
@@ -4,6 +4,6 @@
 HOME_DIR = os.path.dirname(os.path.realpath(__file__)) + "/../../../../"
 
 
-enclave = xgb.Enclave(HOME_DIR + "build/enclave/xgboost_enclave.signed", log_verbosity=0)
-print("Waiting for client...")
-xgb.serve(enclave, all_users=["user1"])
+xgb.init(enclave_image=HOME_DIR + "build/enclave/xgboost_enclave.signed", log_verbosity=0)
+print("Waiting for clients...")
```

Comment:
Revert this back to "Waiting for client..."? In this demo there's only one client.